### DISCUSSION TITLE
Documentation unclear - what's the name of the instance which spins the runners?

### DISCUSSION THREAD

**Discussion by lmilbaum** [npalm/terraform-aws-gitlab-runner#1112] [Q&A]
I've noticed multiple terms which confuses me:
docker machine
gitlab runner
runner worker

I'd like to set the instance type for the instance and for the runners. Can't figure out which keys to set. 
Thanks

**Comment by kayman-mk** [comment]
That is definitely a mess.. I think we need a glossary.

GitLab Runner: also known as Agent. The machine which contacts GitLab and fetches the jobs
Runner Worker: also known as Executor. The machine executing the jobs. Sends the results to the Runner.
docker machine: the driver which is usually installed on the Executor.

Runner configuration is done with `runner_instance`.

**Reply by lmilbaum** [reply]
Thank you so much for the clarification
